<?xml version="1.0"?>

<launch>
  <!-- Openni people segmentation and tracker - adapted to publish the segmented image (does not work upside down, i.e. when robot is watching backwards) (comment out if you do not use the openni-tracker) -->
  <!-- <node pkg="cob_people_detection" ns="cob_people_detection/openni_tracker" type="openni_tracker" name="openniTrackerNode" output="screen"/> -->

  <!-- nodelet manager  (starts somewhere else) -->
  <!-- <node pkg="nodelet" type="nodelet" name="nodelet_manager_people_detection"  args="manager" output="screen"/> -->

  <!-- head detection node (detects faces in depth image and publishes their positions) -->
  <rosparam command="load" ns="cob_people_detection/head_detector" file="$(find cob_people_detection)/ros/launch/head_detector_params.yaml"/>
  <node name="head_detector" pkg="cob_people_detection" ns="cob_people_detection/head_detector" type="head_detector_node" output="screen">
    <remap from="pointcloud_rgb" to="/cam3d/depth/upright/points"/>
	
    <param name="data_directory" type="string" value="$(find cob_people_detection)/common/files/"/>
  </node>

</launch>
